aapl_IEX_model.ipynb

```diff
@@ -0,0 +1,1659 @@
+{
```

Comment:
<p>have <code>common.minimal_IEX_preprocessing(df) </code> return a new  DF, not change in place</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alongouldman/stock_market_neural_nets/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>i do modify the original dataframe, so we have better memory usage.</p><p>ill change the "inplace" argument though.</p>